app/statistics/server/lib/statistics.js

```diff
@@ -74,6 +74,14 @@ export const statistics = {
 		statistics.busyUsers = Meteor.users.find({ status: 'busy' }).count();
 		statistics.totalConnectedUsers = statistics.onlineUsers + statistics.awayUsers;
 		statistics.offlineUsers = statistics.totalUsers - statistics.onlineUsers - statistics.awayUsers - statistics.busyUsers;
+		statistics.languages = { en: 0 };
```

Comment:
AFAIK the default language is not only `en`, the admin can set a different one, you should get that info from the settings.